### PR TITLE
devx/newsletter: May 20, 2022

### PR DESCRIPTION
**Pull request by bobheadxi**
https://deploy-preview-3497--sourcegraph-handbook.netlify.app/departments/product-engineering/engineering/enablement/dev-experience/newsletter/

### CODE REVIEW

File: content/departments/product-engineering/engineering/enablement/dev-experience/newsletter.md [hunk 1]
```diff
@@ -8,6 +8,65 @@ To learn more about components of Sourcegraph's developer experience, check out
...
+
+TODO(@jhchabran)
+
+### Smoke testing ☁️
+
+TODO(@jhchabran)
+
+### Learning resources 🎥
+
+TODO(@marekweb)
```
Comment (bobheadxi) [inline content/departments/product-engineering/engineering/enablement/dev-experience/newsletter.md]: @marekweb

File: content/departments/product-engineering/engineering/enablement/dev-experience/newsletter.md [hunk 2]
```diff
@@ -8,6 +8,65 @@ To learn more about components of Sourcegraph's developer experience, check out
...
+- The output API has been overhauled to be centralized in `std.Out`, which now centralizes the exports of a variety of `sg`-specific utilities for incorporating ✨ _fancy_ ✨ output for some added bling.
+- Writing scripts? We strongly recommend everyone to start writing scripts in Go within `sg`, which gives us more code-sharing opportunities, better cross-platform compatibility, and more advanced features such as better output management. To enable this, the DevX team has started developing a new command execution library, [`github.com/sourcegraph/run`](https://pkg.go.dev/github.com/sourcegraph/run), aimed at providing a seamless way to execute commands and manipulate its output in Go.
+
+### Following your code from PR to production 🚢
+
+TODO(@jhchabran)
+
+### Smoke testing ☁️
+
+TODO(@jhchabran)
```
Comment (bobheadxi) [inline content/departments/product-engineering/engineering/enablement/dev-experience/newsletter.md]: @jhchabran

File: content/departments/product-engineering/engineering/enablement/dev-experience/newsletter.md [hunk 3]
```diff
@@ -8,6 +8,69 @@ To learn more about components of Sourcegraph's developer experience, check out
...
+
+### Learning resources 🎥
+
+TODO(@marekweb)
+
+### TEMP
+
+TODO(@efritz)
```
Comment (efritz) [inline content/departments/product-engineering/engineering/enablement/dev-experience/newsletter.md]:
```suggestion
### Database migration tooling

There are two new goodies for database tooling available via `sg migration` locally and via the `migrator` binary shipped with every Sourcegraph instance.

- New [describe command](https://github.com/sourcegraph/sourcegraph/pull/35641) provides a formatted version of your database's current schema
- New [drift command](https://github.com/sourcegraph/sourcegraph/pull/32472) provides a diff of the expected schema and your database's current schema

And for some bling:

- The new describe command has been [beautified](https://github.com/sourcegraph/sourcegraph/pull/35722)
- The new drift has been [beautified](https://github.com/sourcegraph/sourcegraph/pull/35735)
```

File: content/departments/product-engineering/engineering/enablement/dev-experience/newsletter.md [hunk 4]
```diff
@@ -8,6 +8,115 @@ To learn more about components of Sourcegraph's developer experience, check out
...
+
+### Smoke testing ☁️
+
+Every ten minutes, smoke tests are being run against https://sourcegraph.com, first making basic infrastructure tests then performing a quick search to ensure that the application is up. And it's for real this time, [#16589](https://github.com/sourcegraph/deploy-sourcegraph-cloud/pull/16589) turns failures into an automated incident being created.
+
+### Learning resources 🎥
+
+TODO(@marekweb)
+
+### Tech Radar
```
Comment (jhchabran) [inline content/departments/product-engineering/engineering/enablement/dev-experience/newsletter.md]: cc @vrto

**Pull request merged**